pygmt/modules.py

```diff
@@ -10,48 +10,10 @@
     data_kind,
     dummy_context,
     fmt_docstring,
+    kwargs_to_strings,
     use_alias,
 )
```

Comment:
The whole chunk `from pygmt.helpers import (...)` (9 lines) can be deleted now. Will have to do it manually.

Reply:
Deleted.